### PR TITLE
Issue unique identifier

### PR DESCRIPTION
**Pull request by dancantor**

### CODE REVIEW

File: src/app/store/game-settings.store.ts [hunk 1]
```diff
@@ -11,16 +12,29 @@ export class GameSettingsStore extends Store<GameSettingsState> {
...
+  private getPlayerIdentifier(newPlayerName?: string, playerList?: Player[]) {
+    let playerIdentifier = (newPlayerName?.length ?? 0) > 0
+                            ? newPlayerName![0].toLocaleUpperCase()
+                            : '';
+    let index = 2;
+    // While you still find players with the same identifier
+    while (playerList?.find(element => element.playerIdentifier == playerIdentifier)) {
+      playerIdentifier = newPlayerName!.slice(0, index++); // Add a new letter
+      if (index > (newPlayerName?.length ?? 0) + 1){ // If you've already taken all the possible letters
+        playerIdentifier = '';
```
Comment (vladcantor) [inline src/app/store/game-settings.store.ts:24]: You could start adding numbers to the name

File: src/app/store/game-settings.store.ts [hunk 2]
```diff
@@ -11,16 +12,29 @@ export class GameSettingsStore extends Store<GameSettingsState> {
     super(createInitialState());
   }
 
+  private getPlayerIdentifier(newPlayerName?: string, playerList?: Player[]) {
+    let playerIdentifier = (newPlayerName?.length ?? 0) > 0
+                            ? newPlayerName![0].toLocaleUpperCase()
+                            : '';
```
Comment (vladcantor) [inline src/app/store/game-settings.store.ts:18]: can two players have string empty as identifier?

**Review by vladcantor — COMMENTED** [review]
Great work! Again Congrats!
It would be awesome if you would have time to check the two scenarios i describes below.

File: src/app/store/game-settings.store.ts (hunk 2 above)
Comment (dancantor) [reply src/app/store/game-settings.store.ts:18]: Practically, you can't add the player if it has an empty identifier. So the user need to write a name which results in a possible identifier

**Pull request merged**